File: SceytChatUiKit/src/main/java/com/sceyt/sceytchatuikit/media/audio/FileManager.java
package com.sceyt.sceytchatuikit.media.audio;

import java.io.File;
import java.util.UUID;

public class FileManager {
    public static File createFile(String extension, String directory) {
        File mediaDir = new File(directory);
        if (!mediaDir.exists()) {
            mediaDir.mkdirs();
        }

        return new File(mediaDir, "Audio_" + UUID.randomUUID() + "." + extension);
    }
}

File: SceytChatUiKit/src/main/java/com/sceyt/chatuikit/media/audio/FileManager.kt
package com.sceyt.chatuikit.media.audio

import java.io.File
import java.util.UUID

object FileManager {

    fun createFile(extension: String, directory: String): File {
        val mediaDir = File(directory)
        if (!mediaDir.exists()) {
            mediaDir.mkdirs()
        }
        return File(mediaDir, "Audio_" + UUID.randomUUID() + "." + extension)
    }
}